pkg/morph/contracts/models.go

```diff
@@ -0,0 +1,88 @@
+package fschaincontracts
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+	containerrpc "github.com/nspcc-dev/neofs-contract/rpc/container"
+	"github.com/nspcc-dev/neofs-sdk-go/container"
+	protocontainer "github.com/nspcc-dev/neofs-sdk-go/proto/container"
+	protonetmap "github.com/nspcc-dev/neofs-sdk-go/proto/netmap"
+	"github.com/nspcc-dev/neofs-sdk-go/proto/refs"
+	"github.com/nspcc-dev/neofs-sdk-go/user"
+	"google.golang.org/protobuf/proto"
+)
+
+// ContainerFromStackItem decodes container from stack item.
+func ContainerFromStackItem(item stackitem.Item) (container.Container, error) {
```

Comment:
Why in a separate package? Usually we have struct convertors in the same `morph/client/$contract_name` package and it works fine.